src/localisation/localisation.c

```diff
@@ -1295,3 +1295,75 @@ int win1252_to_utf8(utf8string dst, const char *src, size_t srcLength, size_t ma
 
 	return result;
 }
+
+money32 string_to_money(char * string_to_monetise)
+{
+	const char* decimal_char = language_get_string(STR_LOCALE_DECIMAL_POINT);
+	char * text_ptr = string_to_monetise;
+	int i, j, sign;
+	//Remove everything except numbers decimal, and minus sign(s)
+	for (i = 0; text_ptr[i] != '\0'; ++i) {
+		while (!(
+			(text_ptr[i] >= '0' && text_ptr[i] <= '9') ||
+			(text_ptr[i] == decimal_char[0]) ||
+			(text_ptr[i] == '-') || 
+			(text_ptr[i] == '\0')
+		)) {
+			//move everything over to the left by one
+			for (j = i; text_ptr[j] != '\0'; ++j) {
+				text_ptr[j] = text_ptr[j + 1];
+			}
+			text_ptr[j] = '\0';
+		}
+	}
+	
+	//if first character of shortened string is a minus, consider number negative
+	if (text_ptr[0] == '-') {
+		sign = -1;
+	}
+	//now minus signs can be removed from string
+	for (i = 0; text_ptr[i] != '\0'; ++i) {
+		if (text_ptr[i] == '-') {
+			for (j = i; text_ptr[j] != '\0'; ++j) {
+				text_ptr[j] = text_ptr[j + 1];
+			}
+			text_ptr[j] = '\0';
+		}
+	}
+
+	//determine if decimal exists in text
+	char *decimal_place = strstr(string_to_monetise, decimal_char);
+	if (decimal_place == NULL) {
+		//if decimal char does not exist, keep it basic. multiply by 10 to fill decimal and be done.
+		return atoi(string_to_monetise) * 10;
+	}
+
+	char *tokenize = _strdup(string_to_monetise);
+	char *numberText = strtok(tokenize, decimal_char);
+	char *decimalText = strtok(NULL, decimal_char);
+
+	int number = 0, decimal = 0;
+	if (numberText != NULL) number = atoi(numberText);
+	if (decimalText != NULL) decimal = atoi(decimalText);
+	if (decimal < 10) decimal *= 10; //if less than 10, say, 6, convert to 60.
+
+	free(tokenize);
+
+	return MONEY(number, decimal) * sign;
+}
+
+void money_to_string(money32 amount, char * buffer_to_put_value_to, size_t buffer_len)
```

Comment:
Nah, you have to dig into how the MONEY(fraction, decimal) works, i based it with how the money is set in the console i think. this was wrote many months back so i don't exactly remember where i found the rate of 10 being used, is there something more specific you see i should change to? i could honestly use some help in this portion because the whole money currency and exchange rate is a bit complicated for me to grasp.